pkg/search/options.go

```diff
@@ -72,7 +72,7 @@ var (
 	ImageName                     = newFieldLabel("Image")
 	ImageSHA                      = newFieldLabel("Image Sha")
 	ImageSignatureFetchedTime     = newFieldLabel("Image Signature Fetched Time")
-	ImageSignatureVerified        = newFieldLabel("Image Signature Verified")
+	ImageSignatureVerifiedBy      = newFieldLabel("Image Signature Verified By")
```

Comment:
Does this change require UI changes?

When creating a policy with the `Trusted Image signers` criteria, if I view the policy after creation, the field name will be empty:
![Screenshot 2022-04-05 at 04 15 30](https://user-images.githubusercontent.com/89904305/161665945-b98f8233-87da-4aed-9689-226327ded86a.png)

Is there something we need to change in the `descriptors.tsx` file? Maybe @ivan-degtiarenko knows more.